grammar.js

```diff
@@ -802,12 +802,18 @@ module.exports = grammar({
       '{',
       repeat(seq(
         optional('static'),
-        $.method_definition,
+        choice(
+          $.method_definition,
+          $._public_field_definition
+        ),
         optional(';')
       )),
       '}'
     ),
 
+    // TODO this should be a property_name
```

Comment:
With this TODO, do you mean that we should rename this thing? Or are you talking about a bigger restructuring?